### PR TITLE
add hooks helm-swoop-goto-line-action-hook and helm-swoop-before-goto-line-action-hook

### PR DESCRIPTION
**Pull request by jixiuf**
#101
```
(defun push-mark-before-helm-swoop()
  (bm-bookmark-add nil nil t)             ; push bookmark for bm.el
  )
(add-hook 'helm-swoop-before-goto-line-action-hook 'push-mark-before-helm-swoop)

demo for evil user:
(defun push-mark-before-helm-swoop()
  (isearch-update-ring helm-pattern evil-regexp-search))
(add-hook 'helm-swoop-after-goto-line-action-hook 'push-mark-before-helm-swoop)
```

### CODE REVIEW

File: helm-swoop.el [hunk 1]
```diff
@@ -150,6 +150,15 @@
     (switch-to-buffer $buf))
   "Change the way to split window only when `helm-swoop' is calling")
 
+(defcustom helm-swoop-goto-line-action-hook nil
```
Comment (alphapapa) [inline helm-swoop.el]: For consistency, this should probably be called `helm-swoop-after-goto-line-action-hook`.  :)

**Comment by alphapapa** [comment]
Looks like a great addition, can this be merged?

File: helm-swoop.el (hunk 1 above)
Comment (jixiuf) [inline helm-swoop.el]: now  renamed to helm-swoop-after-goto-line-action-hook

**Comment by ShingoFukuyama** [comment]
Thank you for the nice suggestion! It's merged.

**Pull request merged**